### PR TITLE
CC-5904 Removes bower dependency.

### PR DESCRIPTION
**Pull request by qubyte**
@BrandwatchLtd/vizia-dev Ready for review.

### CODE REVIEW

File: test/test.js [hunk 1]
```diff
@@ -0,0 +1,232 @@
+mocha.ui('bdd');
+mocha.reporter('html');
+
+describe('hermes', function () {
+    'use strict';
+
+    var assert = chai.assert;
+
+    describe('setup', function () {
+        it('should attach a list to the provided element', function () {
```
Comment (cham) [inline test/test.js]: I think we have some rule about not using "should" in tests, since it's not very valuable

Comment (cham) [inline test/test.js]: e.g.: it looks like all your tests start with "should" - you can just remove that and not lose any value e.g.: 'should attach a list' -> 'attaches a list'

**Comment by cham** [comment]
just that 'should' thing, otherwise LGTM :+1:

File: test/test.js (hunk 1 above)
Comment (qubyte) [inline test/test.js]: This is just moved code, but I can update.

**Comment by lcapel** [comment]
Looking good, just a question, if the bower dependency is gone, why do you still need a bower.json file?

**Comment by qubyte** [comment]
@lcapel For users consuming this module via bower.

**Comment by qubyte** [comment]
@cham & @lcapel Comments addressed.

**Comment by lcapel** [comment]
:+1:

**Comment by cham** [comment]
:+1:

**Pull request merged**